src/briefcase/commands/create.py

```diff
@@ -686,8 +686,8 @@ def install_app_resources(self, app: AppConfig):
         # Briefcase v0.3.18 - splash screens deprecated.
         if getattr(app, "splash", None):
             self.logger.warning(
-                "Splash screens are now configured based on the icon. "
-                "The splash configuration will be ignored."
+                "\nSplash screens are now configured based on the icon. "
+                "The splash configuration will be ignored.\n"
             )
```

Comment:
As an FYI, as a consequence of `str.splitlines()` in `Log()`, a single trailing newline is effectively ignored. However, a trailing newline in the output text can be achieved with two trailing newlines in the string.

```pycon
>>> "\nline of text\n".splitlines()
['', 'line of text']
>>> "\nline of text\n\n".splitlines()
['', 'line of text', '']
```
Same as the "extra" newline in the multiline strings.
```pycon
>>> """line 1
... line 2
... line 3
... 
... """
'line 1\nline 2\nline 3\n\n'
```

Reply:
Good catch - I'd completely forgotten we'd made this change. I've cleaned this up in #1769.

It would be nice if there was a way to automate catching stylistic regressions like this, but I'm not sure I have any ideas on how we'd do that.